--- product.json	2019-03-15 05:31:32.000000000 +0700
+++ product.json_new	2019-03-16 00:58:14.614098472 +0700
@@ -20,5 +20,11 @@
 	"urlProtocol": "code-oss",
 	"extensionAllowedProposedApi": [
 		"ms-vscode.references-view"
-	]
+	],
+	"quality": "stable",
+	"extensionsGallery": {
+  	    "serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
+  	    "cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
+  	    "itemUrl": "https://marketplace.visualstudio.com/items"
+	}
 }
